### ISSUE TITLE
Some missing usability features/keybinds.

### ISSUE THREAD

**Issue by that1communist** [liri-project/liri-browser#141]
1. You can't scroll on the tab bar to change through which tab is selected
2. ctrl+L doesn't select the url bar
3. The mouse doesn't change to the... for lack of a better term, clicky hand, when over something you can click.
4. when you click on a on/off setting and not the actual switch for the setting, it plays the material effect, but doesn't flip the switch.

**Comment by tim-sueberkrueb** [comment]
Thanks for reporting!
1. This should work since v0.3
2. Thanks, fixed that!
3. There's already an issue for that: #32
4.Thanks, fixed that!
Can you explain more precisely what you mean with 1?

**Comment by VirusKA** [comment]
lol clicky hand.
but yeah that's been an issue for a while. Shady code?

**Comment by that1communist** [comment]
On chrome, if you use the scroll wheel on the tab bar, it'll go through them, that's handy as hell, this just moves them back\forth.

**Comment by VirusKA** [comment]
yeah we should add that

**Comment by tim-sueberkrueb** [comment]
@that1communist Right, but chrome also makes the tabs smaller the more are opened. Liri hides (as Firefox does) the tabs and you can scroll (or flick if you're using a touch screen) through them.
I prefer this way. Might be possible in combination with #109.
